### DISCUSSION TITLE
Windows x86 build fails security check

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [terraform-linters/tflint#1699] [General]
### Summary

After downloading the zip file for x86 build `[tflint_windows_386.zip](https://github.com/terraform-linters/tflint/releases/download/v0.45.0/tflint_windows_386.zip)` windows flagging as trojan. 

[https://imgur.com/a/wEnZs2V](https://imgur.com/a/wEnZs2V
) 

### Command

...

### Terraform Configuration

```terraform
...
```

### TFLint Configuration

```HCL
...
```

### Output

```Shell
...
```

### TFLint Version

0.45.0

### Operating System

- [ ] Linux
- [ ] macOS
- [X] Windows

**Comment by bendrucker** [comment]
Almost assuredly not a bug. If there's something we can do about this feel free to share.

https://www.google.com/search?q=wacatac.b!ml+false+positive+site:answers.microsoft.com&client=safari&rls=en&sxsrf=AJOqlzVNaRltRZ-lhHtxhabiQ6ITju5dZg:1677186500042&sa=X&ved=2ahUKEwj1jrrMxqz9AhUoHUQIHRsTD9AQrQIoBHoECBYQBQ&biw=2338&bih=1363&dpr=2

**Reply by kdpuvvadi** [reply]
Can we submit the file for review? And why only x86 is flagged? And not the previous versions?

**Reply by bendrucker** [reply]
Who is we? What is review? The TFLint maintainers are not going to spend time on this without a clear indication that the project is somehow at fault, but feel free to pursue additional details and report back.

**Reply by kdpuvvadi** [reply]
I mean to Microsoft

**Reply by bendrucker** [reply]
We're not Windows users and therefore can't reproduce this for a submission. We just publish Windows binaries and use Actions VMs to test on Windows. If this is a process you're familiar with, we don't need to have a vague back and forth.

Go ahead and submit it and report back:

https://www.microsoft.com/en-us/wdsi/filesubmission

As a volunteer-maintained project, we don't have the time or energy to get involved in support cases with Microsoft and help them fix their buggy malware detection. In the unlikely event that the build is genuinely compromised and you can come back with a clear report, obviously we'll act.

**Reply by kdpuvvadi** [reply]
seems like there is no issue with it.

![image](https://user-images.githubusercontent.com/7326255/221367182-ba03a69e-cdd8-46cf-8735-5728b8968042.png)

**Reply by bendrucker** [reply]
Great, thanks for verifying
